### PR TITLE
Solution to leetcode486

### PR DESCRIPTION
**Pull request by janicelmm**
Solution to leetcode486

### CODE REVIEW

File: src/leetcode435NonoverlappingIntervals.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import java.util.Arrays;
+import java.util.Comparator;
+import java.util.PriorityQueue;
```
Comment (Ewlai) [inline src/leetcode435NonoverlappingIntervals.java]: These are unused imports

File: src/leetcode435NonoverlappingIntervals.java [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+            else
+            {
+                tail = intervals[i][1];
+            }
+        }
+        
+        return count;
+    }
+    public static void main(String[] args) {
+        // TODO Auto-generated method stub
```
Comment (Ewlai) [inline src/leetcode435NonoverlappingIntervals.java]: Todo comments are a reminder there is further code to write.  This is no longer the case

**Review by Ewlai — CHANGES_REQUESTED** [review]
Overall good solution. Some minor housekeeping things to be done.

File: src/leetcode435NonoverlappingIntervals.java (hunk 1 above)
Comment (janicelmm) [reply src/leetcode435NonoverlappingIntervals.java]: Updated.

File: src/leetcode435NonoverlappingIntervals.java (hunk 2 above)
Comment (janicelmm) [reply src/leetcode435NonoverlappingIntervals.java]: Updated.

**Review by janicelmm — COMMENTED** [review]
Updated based on comments

**Review by Ewlai — APPROVED** [review]
LGTM

**Pull request merged**